.pipelines/build-model-container-image.Jenkinsfile

```diff
@@ -0,0 +1,95 @@
+pipeline {
+    agent { label 'master' }
+    environment {
+        ML_IMAGE_FOLDER = 'imagefiles'
+        IMAGE_NAME      = 'mlmodelimage'
+        MODEL_NAME      = "${MODEL_NAME}"
+        SCORE_SCRIPT    = 'scoring/score.py'
+        RESOURCE_GROUP  = "${RESOURCE_GROUP}"
+        WORKSPACE_NAME  = "${WORKSPACE_NAME}"
+        ML_CONTAINER_REGISTRY   = "${ML_CONTAINER_REGISTRY}"
+    }
+    stages {
+        stage('initialize') {
+            steps {
+                echo 'Remove the previous one!'
+            }
+            post {
+                always {
+                    deleteDir() /* clean up our workspace */
+                }
+            }
+        }
+        stage ('Download build artifacts (get model name)') {
+            environment {
+                BUILD_ARTIFACT_FOLDER = "download"
+            }
+            steps {
+                script {
+                    copyArtifacts(projectName: "${env.build_job_name}", selector: buildParameter("ml_model_selector"), target: "${env.BUILD_ARTIFACT_FOLDER}");
+                    def  FILES_LIST = sh (script: "ls   '${env.BUILD_ARTIFACT_FOLDER}'", returnStdout: true).trim()
+                    //DEBUG
+                    echo "FILES_LIST : ${FILES_LIST}"
+                    MODEL_NAME = sh (script: "cat ${env.BUILD_ARTIFACT_FOLDER}/model_name.txt", 
+                                     returnStdout: true).trim()
+                    sh "echo ${MODEL_NAME}"
+                }
+            }
+        }
+        stage('generate_dockerfile') {
+            steps {
+                echo "Hello docker image build ${env.BUILD_ID}"
+                checkout scm
+                //checkout([$class: 'GitSCM', branches: [[name: '*/ml_model_uc81']],
+                //    userRemoteConfigs: [[url: 'https://github.com/Merlion-Crew/MLOpsPython.git/']]])
+
+                sh '''
+                    conda env create --file ./diabetes_regression/ci_dependencies.yml --force 
+                '''
+                
+                withCredentials([azureServicePrincipal("${AZURE_SP}")]) {
+                    sh '''#!/bin/bash -ex
+                        az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET -t $AZURE_TENANT_ID
+                        az account set -s $AZURE_SUBSCRIPTION_ID
+                        export SUBSCRIPTION_ID=$AZURE_SUBSCRIPTION_ID
+                        source /home/azureuser/anaconda3/bin/activate mlopspython_ci
```

Comment:
mlopspython_ci is the name inside the ci_dependencies.yml file right? that you are running in Line 47?
Might be worth looking into parameterizing it (may be as a cleanup PR after this). Reason I am suggesting this is if in the future someone changes the environment name in ci_dependencies.yml then they should also be aware to update this line in this pipeline - not ideal for maintainability purpose. 

Suggest you add a task and clean it up in the subsequent PR.